hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/HTSPrecompiledContract.java

```diff
@@ -225,26 +216,64 @@ protected Bytes computeAssociateTokens(final Bytes input, final MessageFrame mes
 		return null;
 	}
 
-	@SuppressWarnings("unused")
+	//	TODO: Abstract out the invocation of the ledgers as a helper method to avoid duplication
+	//	TODO: Properly instantiate syntheticTxn after Mr.Thinker provides implementation
+	//	TODO: It seems that the ledgers.commit() call duplicates commit calls in the logic.associate() method.
+	//	TODO: Clarify - are the customFees those, contained in the Token's List<FcCustomFee> customFees property.
```

Comment:
I think for the associate and dissociate implementations we can use an empty list of `FcAssessedCustomFee`